pandas/tests/io/test_parquet.py

```diff
@@ -707,6 +707,14 @@ def test_duplicate_columns(self, pa):
         df = pd.DataFrame(np.arange(12).reshape(4, 3), columns=list("aaa")).copy()
         self.check_error_on_write(df, pa, ValueError, "Duplicate column names found")
 
+    def test_write_column_fp16(self, pa):
```

Comment:
def test_unsupported_float16

Reply:
Thanks, I have changed the name of the test in the latest commit.